sdk/core/azure-core/azure/core/utils/_pipeline_transport_rest_shared.py

```diff
@@ -333,7 +334,7 @@ def parse_responses(response):
     return responses
 
 
-def _format_data_helper(data: Union[str, IO]) -> Union[Tuple[None, str], Tuple[Optional[str], IO, str]]:
+def _format_data_helper(data: FileType) -> Union[Tuple[None, str], Tuple[Optional[str], FileContent, str]]:
```

Comment:
the previous typing of the file just being `IO` is currently incorrect, because we have always specified that [`FileContent`](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/core/azure-core/azure/core/rest/_helpers.py#L68) can be bytes, string, or io of bytes or string